recipes/rocksdb/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+### Build
+export INSTALL_PATH=${PREFIX}
+make release
+make shared_lib
```

Comment:
Is there a `make check` or similar that we can run? Also would there be any value in building the static library as well?